test/Microsoft.AspNet.Authentication.Test/Cookies/CookieMiddlewareTests.cs

```diff
@@ -421,6 +421,27 @@ private Task SignOutAsWrong(HttpContext context)
         }
 
         [Fact]
+        public async Task CookieTurns401ToAccessDeniedWhenSetAndIfAuthenticated()
+        {
+            var clock = new TestClock();
+            TestServer server = CreateServer(options =>
+            {
+                options.SystemClock = clock;
+                options.AccessDeniedPath = new PathString("/accessdenied");
```

Comment:
That test already existed, https://github.com/aspnet/Security/blob/3-3accessdenied/test/Microsoft.AspNet.Authentication.Test/Cookies/CookieMiddlewareTests.cs#L406